api/controllers/swagger.go

```diff
@@ -296,6 +302,58 @@ func (s *SwaggerController) ServeSwaggerSpec(c *fireball.Context) (fireball.Resp
 					},
 				},
 			},
+			"/deploy": map[string]swagger.Method{
+				"get": {
+					Summary: "List all Deploys",
+					Tags:    []string{"Deploy"},
+					Responses: map[string]swagger.Response{
+						"200": {
+							Description: "An array of deploys",
+							Schema:      swagger.NewObjectSliceSchema("Deploy"),
+						},
+					},
+				},
+				"post": {
+					Summary: "Add a Deploy",
+					Tags:    []string{"Deploy"},
+					Parameters: []swagger.Parameter{
+						swagger.NewBodyParam("CreateDeployRequest", "Task to add", true),
+					},
+					Responses: map[string]swagger.Response{
+						"200": {
+							Description: "The added task",
+							Schema:      swagger.NewObjectSchema("Task"),
+						},
+					},
+				},
+			},
+			"/deploy/{id}": map[string]swagger.Method{
+				"get": {
+					Summary: "Describe a Deploy",
+					Tags:    []string{"Task"},
```

Comment:
This needs to be `Tags:    []string{"Deploy"},` otherwise this will appear in the Task grouping of methods in swagger.